locales/en-US/app.ftl

```diff
@@ -12,9 +12,16 @@ footerLinkPrivacy = Privacy
 footerLinkTerms = Terms
 footerLinkLegal = Legal
 footerLinkFeedback = Give Feedback
-footerLinkAbout = About Test Pilot
-
-## Items in the menu.
+footerLinkAboutHeader = About
+footerLinkAboutUs = About us
+footerLinkSupportHeader = Support
+footerLinkDownload = Download Firefox
+footerLinkMobile = Mobile
```

Comment:
I assume this switches to the mobile version of the website? Can we add a comment to explain it?

Reply:
This links to the site where you can download mobile firefox, android or ios

Reply:
OK, definitely worth a comment then.